sql_generators/active_users_aggregates_v4/templates/mobile_schema.yaml

```diff
@@ -0,0 +1,93 @@
+fields:
```

Comment:
The desktop v4 schema contains attribution columns while the mobile schema doesn't; is that intentional?